daemon/prune.go

```
@@ -190,42 +190,39 @@ func (daemon *Daemon) ImagesPrune(pruneFilters filters.Args) (*types.ImagesPrune
 }
 
 // localNetworksPrune removes unused local networks
-func (daemon *Daemon) localNetworksPrune(pruneFilters filters.Args) (*types.NetworksPruneReport, error) {
+func (daemon *Daemon) localNetworksPrune(pruneFilters filters.Args) *types.NetworksPruneReport {
 	rep := &types.NetworksPruneReport{}
 
-	until, err := getUntilFromPruneFilters(pruneFilters)
-	if err != nil {
-		return rep, err
-	}
+	until, _ := getUntilFromPruneFilters(pruneFilters)
```

Why ignoring the error here ? 💦